src/goMain.ts

```diff
@@ -158,6 +159,7 @@ export function activate(ctx: vscode.ExtensionContext): void {
 			registerCompletionProvider(ctx);
 			ctx.subscriptions.push(vscode.languages.registerHoverProvider(GO_MODE, new GoHoverProvider()));
 			ctx.subscriptions.push(vscode.languages.registerDefinitionProvider(GO_MODE, new GoDefinitionProvider()));
+			ctx.subscriptions.push(vscode.languages.registerTypeDefinitionProvider(GO_MODE, new GoTypeDefinitionProvider()));
```

Comment:
We need this feature even when the language server is enabled. So. move this line down near line 182

Reply:
You right! Done.